spark/src/main/scala/geotrellis/spark/ingest/package.scala

```diff
@@ -10,26 +10,15 @@ import monocle.syntax._
 
 
 package object ingest {
-  type ProjectedExtentComponent[T] = KeyComponent[T, ProjectedExtent]
-
-  implicit class ProjectedExtentComponentMethods[T: ProjectedExtentComponent](key: T) {
-    val _projectedExtent = implicitly[ProjectedExtentComponent[T]]
-
-    def projectedExtent: ProjectedExtent = key &|-> _projectedExtent.lens get
-
-    def updateProjectedExtent(pe: ProjectedExtent): T =
-      key &|-> _projectedExtent.lens set (pe)
-  }
-
-  implicit object ProjectedExtentComponent extends IdentityComponent[ProjectedExtent]
-
-  implicit class withCollectMetadataMethods[K: ProjectedExtentComponent, V <: CellGrid](rdd: RDD[(K, V)]) extends Serializable {
-    def collectMetaData(crs: CRS, layoutScheme: LayoutScheme): (Int, RasterMetaData) = {
-      RasterMetaData.fromRdd(rdd, crs, layoutScheme)(_.projectedExtent.extent)
+  implicit class withCollectMetadataMethods[K1, V <: CellGrid](rdd: RDD[(K1, V)]) extends Serializable {
```

Comment:
I think we can put this in `geotrellis.spark` package so `ingest` import is not required.